stable/insights-admission/README.md

```diff
@@ -0,0 +1,84 @@
+# Fairwinds Insights Admission Controller
+
+## Intro
+
+This is an Admission Controller to reject deployment of objects into your Kubernetes cluster that woudld create dangerous action items in [Fairwinds Insights](https://insights.fairwinds.com)
+
+## Installation
+```bash
+helm repo add fairwinds-stable https://charts.fairwinds.com/stable
+helm install gemini fairwinds-stable/insights-admission \
+  --namespace insights-admission \
+  --set insights.organization=acme-co \
+  --set insights.cluster=staging \
+  --set insights.base64token="dG9rZW4="
+```
+
+## Chart Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` |  |
+| autoscaling.enabled | bool | `false` |  |
+| autoscaling.maxReplicas | int | `100` |  |
+| autoscaling.minReplicas | int | `1` |  |
+| autoscaling.targetCPUUtilizationPercentage | int | `80` |  |
+| certificateupdater.image.repository | string | `"newrelic/k8s-webhook-cert-manager"` |  |
+| certificateupdater.image.tag | string | `"1.3.0"` |  |
+| certificateupdater.resources.limits.cpu | string | `"150m"` |  |
+| certificateupdater.resources.limits.memory | string | `"512Mi"` |  |
+| certificateupdater.resources.requests.cpu | string | `"100m"` |  |
+| certificateupdater.resources.requests.memory | string | `"128Mi"` |  |
+| fullnameOverride | string | `""` |  |
+| image.pullPolicy | string | `"Always"` |  |
+| image.repository | string | `"quay.io/fairwinds/insights-admission-controller"` |  |
+| image.tag | string | `"0.1"` |  |
+| imagePullSecrets | list | `[]` |  |
+| insights.base64token | string | `""` |  |
+| insights.cluster | string | `""` |  |
+| insights.host | string | `"https://insights.fairwinds.com"` |  |
+| insights.organization | string | `""` |  |
+| nameOverride | string | `""` |  |
+| nodeSelector | object | `{}` |  |
+| podAnnotations | object | `{}` |  |
+| podSecurityContext | object | `{}` |  |
+| replicaCount | int | `1` |  |
+| resources.limits.cpu | int | `1` |  |
+| resources.limits.memory | string | `"2Gi"` |  |
+| resources.requests.cpu | string | `"100m"` |  |
+| resources.requests.memory | string | `"128Mi"` |  |
+| rules[0].apiGroups[0] | string | `"apps"` |  |
```

Comment:
If you add some comments to `values.yaml`, it'll clean this up a bit. Goldilocks is a great example.

Reply:
Thanks!